runtime/core/portable_type/c10/c10/util/irange.h

```diff
@@ -0,0 +1,123 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
+
+#include <c10/util/TypeSafeSignMath.h>
+
+#include <algorithm>
```

Comment:
does it need `<algorithm>`?

Reply:
not relevant; this is a direct copy from pytorch core.